junwoo/db/migrations/20230126032441_create_posts_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE posts(
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+    title VARCHAR(100) NOT NULL,
+    content VARCHAR(2000) NULL,
+    image_url VARCHAR(1000) NULL,
+   	user_id INT NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+    );
```

Comment:
- sql 파일도 마찬가지로 들여쓰기 혹은 포매팅이 잘못되지 않았나 한번 더 확인하는 습관을 들여주세요!
이런 사소한 컨벤션이 개발하는 프로젝트, 더 나아가 개발팀의 문화가 될 수 있습니다.

- 추가로 `user_id` 컬럼은 users 테이블을 참조하고 있는 외래키 역할을 할텐데요, 이에 대한 제약조건을 추가해주세요!
해당 파일로 작성하지 않고, sql 파일을 하나 더 추가해주세요!

Hint))
```
ALTER TABLE users ADD ... 
```


```suggestion
CREATE TABLE posts(
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY, 
    title VARCHAR(100) NOT NULL,
    content VARCHAR(2000) NULL,
    image_url VARCHAR(1000) NULL,
    user_id INT NOT NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
);
```